services/editor/src/pages/settings/components/HooksPage/EditHookPage.js

```diff
@@ -1,5 +1,6 @@
 import React, { useState, useMemo, useContext, useCallback } from 'react';
 import qs from 'query-string';
+import { WithContext as ReactTags } from 'react-tag-input';
```

Comment:
looks good, how come it's WithContext as ReactTags

Reply:
I copied it from the key page